packages/sockethub/src/resource-manager.ts

```diff
@@ -33,6 +33,7 @@ function resourceManagerCycle() {
 
       if (platformInstance.global) {
         // static platform for global use, don't do resource management
+        return;
```

Comment:
@silverbucket Why did you use `return` instead of `continue` now? That will stop the whole `for` loop that iterates over all platform instances. Is that really intended?

Reply:
@galfert Thanks for pointing this out, I was a little distracted. It's fixed in master